docs/notebook-demos/reading-in-sentinel-data.ipynb

```diff
@@ -0,0 +1,225 @@
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "jp2s = [\"/tmp/B01.jp2\", \"/tmp/B09.jp2\", \"/tmp/B10.jp2\"]\n",
+    "arrs = []\n",
+    "\n",
+    "for jp2 in jp2s:\n",
+    "    with rasterio.open(jp2) as f:\n",
+    "        arrs.append(f.read(1))\n",
+    "\n",
+    "data = np.array(arrs, dtype=arrs[0].dtype)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating the RDD\n",
+    "\n",
+    "With our raster data in hand, we can how begin the creation of a Python `RDD`. Please see the [core concepts](core-concepts.ipynb) guide for more information on what following instances represent."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Create an Extent instance from rasterio's bounds\n",
+    "extent = gps.Extent(*f.bounds)\n",
+    "\n",
+    "# The EPSG code can also be obtained from the information read in via rasterio\n",
+    "projected_extent = gps.ProjectedExtent(extent=extent, epsg=int(f.crs.to_dict()['init'][5:]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# We can create a Tile instance from our multiband, raster array and the nodata value from rasterio\n",
+    "tile = gps.Tile.from_numpy_array(numpy_array=data, no_data_value=f.nodata)\n",
+    "tile"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Now that we have our ProjectedExtent and Tile, we can create our RDD from them\n",
+    "rdd = pysc.parallelize([(projected_extent, tile)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating the Layer\n",
+    "\n",
+    "From the `RDD`, we can now create a `RasterLayer` using the `from_numpy_rdd` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# While there is a time component to the data, this was ignored for this tutorial and instead the focus is just\n",
+    "# on the spatial information. Thus, we have a LayerType of SPATIAL.\n",
+    "raster_layer = gps.RasterLayer.from_numpy_rdd(layer_type=gps.LayerType.SPATIAL, numpy_rdd=rdd)\n",
+    "raster_layer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Where to Go From Here\n",
+    "\n",
+    "By creating a `RasterLayer`, we can now work with and analyze the data within it. If you wish to know more about these operations, please see the following guides: [RasterLayer](raster-layer.ipynb), [TiledRasterLayer](tiled-raster-layer.ipynb), [operations-guide], [visulation-guide], and [catalog-guide]."
```

Comment:
Oh, those aren't supposed to link to anything. I just put them there to remind myself to add them once those other guides are done.